ci/prow/e2e-crd-variable

```diff
@@ -2,13 +2,12 @@
 
 set -euxo pipefail
 
-check_kmm_mod () {
-oc debug node/$NODE -T -- chroot /host sh -c "lsmod | grep kmm"
-}
-
 echo "Get first node..."
 NODE=$(oc get nodes -o jsonpath='{.items..metadata.name}' | awk {'print $1'})
 
+echo "Load check_kmm_kmod function..."
+source check_kmm_kmod
```

Comment:
This is only "combining the new files", you still need to use the function somewhere.
Also can we source `check-kmm-kmod` at the top of all files?

Reply:
Same comment for all files.

Reply:
Function name did not change so it is still called.